keps/sig-network/2030-topology-aware-proxying/README.md

```diff
@@ -245,6 +253,138 @@ not reliant on any other feature being enabled in any other release. [KEP
 #2004](https://github.com/kubernetes/enhancements/issues/2004) will be dependent
 on this KEP though.
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: TopologyAwareProxying
+    - Components depending on the feature gate: kube-proxy (also used by KEP
+      2004 for kube-controller-manager)
+
+* **Does enabling the feature change any default behavior?**
+  Not really. Kube-Proxy will still consume endpoints without these topology
+  labels (the default). When these labels are set (a separate opt-in discussed
+  in KEP 2004), kube-proxy will only consume a subset of endpoints closest to
+  it. This will also not be noticeable in most cases.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Yes, disabling the feature gate or simply disabling this functionality per
+  Service (KEP 2004) will be supported.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  A potentially smaller set of endpoints may be routed to if this feature was
+  also enabled as described in KEP 2004.
```

Comment:
Let's say it is - does reprogramming iptables can break any of the existing connections?

Reply:
no, existing connections will be unaffected.